### ISSUE TITLE
setup_post_meta() queries duplicated

### ISSUE THREAD

**Issue by jimboc100** [wpninjas/ninja-forms#3324]
In a list of posts, there are duplicate queries running on each post.

The query looks like:
SELECT `meta_key`, `meta_value` FROM dts_11_postmeta WHERE `post_id` = ####

The callers are:
NF_MergeTags_Deprecated->setup_post_meta()
NF_MergeTags_WP->setup_post_meta()

Why is Deprecated running at the same time as WP?

**Comment by paulgibbs** [comment]
Why exactly does the plugin need to make an uncached SQL query for all post meta for each post in WP's template loop? It guarantees worse performance the more posts you have in a loop, the more post meta you have attached to a post.

**Comment by jimboc100** [comment]
First we make it work. Then the server CPU peaks out. Then we make it work well.

You are correct, this is not what I want at all. I am making an attempt to eliminate this issue. But I did not write the initial code base.

See the screenshot for the list of queries running that prompted the question.

<img width="1343" alt="screen shot 2018-01-30 at 10 13 35 am" src="https://user-images.githubusercontent.com/6885629/35577154-71ac8490-05a6-11e8-8424-d7ac33ac63ef.png">

**Comment by paulgibbs** [comment]
Right, I didn't mean to imply you were the original author. I've just found similar issues on a site I was developing and saw your recent Github issue (this one), so figured I'd chime in. :)

**Comment by jimboc100** [comment]
No offense taken. It is what it is and now I have to clean it up. I am just at a loss as to how to do so.

Kinda glad it's not just me. :)

**Comment by jimboc100** [comment]
Just upgraded to 3.2.11--same problem.
